### PR TITLE
Add instructions for AMP-AD

### LINKED ISSUE #161: Include process instructions
Copy/pasted from Bug Bash document.
>Include language that describes the process going from submitting study and assay documentation to submitting manifest and metadata files. Could be at a button next to ‘Submit’ called ‘further instructions’, something like “contact ….. for information about manifest parent ID’

Mette will draft the suggested language.

### PR DESCRIPTION
**Pull request by karawoo**
Closes #161. Provides more thorough documentation of the full process of submitting data in AMP-AD. @Aryllen made a flow chart that could be added to this as well. @amapeters would you please take a look? If it's easier to view all the text at once, you can see it [here](https://github.com/Sage-Bionetworks/dccvalidator/blob/698460ced0f5b0b22ff7867929dddca333d778f4/vignettes/using-the-dccvalidator-app-amp-ad.Rmd).

### CODE REVIEW

**Comment by Aryllen** [comment]
I have a question regarding Step 5:
https://github.com/Sage-Bionetworks/dccvalidator/blob/698460ced0f5b0b22ff7867929dddca333d778f4/vignettes/using-the-dccvalidator-app-amp-ad.Rmd#L113-L117

Should the contributors be contacting you to gain access for writing to the folder once validated? If they have access from the start of the process, there would be nothing to stop them from uploading data that fails validation or even without validating at all.

**Comment by kelshmo** [comment]
@Aryllen our existing model is for contributors to gain access for writing to the folder only after they have passed validation. The downside of this model is we can be a blocker for contributors but the upside is we can gate upload to ensure higher quality! I tend to give access to a metadata folder and not the data folder, but may modify this approach moving forward since the app uploads on the backend. If this is how you want to proceed with AMP, I suggest adding a direction to contact DCC when validation is complete to gain access to the folder. (For existing studies we may have to remove access permissions?)

**Comment by karawoo** [comment]
@Aryllen will you take a look at this when you get a chance? I've coordinated with Mette about it offline and updated things to reflect her comments.

File: vignettes/using-the-dccvalidator-app-amp-ad.Rmd [hunk 1]
```diff
@@ -38,52 +31,96 @@ Some portions of the app submit data to Synapse. This allows curators at Sage to
...
+can upload multiple files to the assay description section. 
+![A screenshot of a website with instructions on how to provide documentation of a study, and a space to upload documentation files](../man/figures/amp-documentation.png).
 
-Each study should include metadata that would help a new researcher understand
-and reuse the data. In most cases, we will expect 4 files:
+If you are submitting documentation for multiple studies, it should be submitted
+one study at a time. To acknowledge your data contribution you can upload an
+acknowledgement statement, and/or publication that should be referenced in
+publications based on secondary data use, together with the study and assay
+documentation.
```
Comment (Aryllen) [inline vignettes/using-the-dccvalidator-app-amp-ad.Rmd]: The acknowledgement should be within one of the files, either study documentation or one of the assay files. I think the way this reads might be a little confusing. It sounds as if there should be a separate file for the acknowledgement, but we don't provide a separate upload input for that. It also sounds as if they should upload the actual publication they want referenced.

>To acknowledge your data contribution you can add an
acknowledgement statement, and/or a preferred publication citation that should be referenced in
publications based on secondary data use, within the study or assay
documentation.

File: vignettes/using-the-dccvalidator-app-amp-ad.Rmd [hunk 2]
```diff
@@ -16,17 +16,10 @@ knitr::opts_chunk$set(
...
-that it will help you, the data contributor, get your data checked, validated,
-and shared more easily and quickly.
-
-The application can be found at https://shinypro.synapse.org/users/kwoo/dccvalidator-app/
-
-# Instructions
+The dccvalidator tool streamlines the data curation process by performing
+metadata validation checks before the data is uploaded. The dccvalidator
+application can be found at
+https://shinypro.synapse.org/users/kwoo/dccvalidator-app/
```
Comment (Aryllen) [inline vignettes/using-the-dccvalidator-app-amp-ad.Rmd]:
```suggestion
https://shinypro.synapse.org/users/kwoo/dccvalidator-app/.
```

**Pull request merged**